.gitmodules

```diff
@@ -1,12 +1,14 @@
 [submodule "nettle"]
 	path = firmware/nettle
-	url = ../nettle.git
+	url = https://github.com/nthdimtech/nettle.git
+	ignore = dirty
 [submodule "crosstool-ng"]
-	path = firmware/crosstool-ng
+	path = crosstool-ng
 	url = https://github.com/crosstool-ng/crosstool-ng.git
 [submodule "firmware-hc/nettle"]
 	path = firmware-hc/external/nettle
-	url = ../nettle.git
+	url = https://github.com/nthdimtech/nettle.git
```

Comment:
See previous comment about nettle library

Reply:
See previous answer :)

Moreover, full URLs are no limitation. If I really want to "mirror" the whole thing, I'll need to fork every repo anyway.

Reply:
Right but you'd need to change your module paths in .gitmodules or else the mirror/fork doesn't stand by itself. I realize this is a bit of a picky point but it comes from the realization that hosting companies gain a lot of power by taking advantage of the fact that projects tend to build in assumptions into their process about how it is hosted. Then there later comes a time where a company becomes a bad actor and there is resistance in the community because of perceived work or risk of a migration.

Reply:
I see, you have a point.